torch/csrc/utils/tuple_parser.cpp

```diff
@@ -63,6 +63,22 @@ auto TupleParser::parse(std::vector<int>& x, const std::string& param_name) -> v
   }
 }
 
+auto TupleParser::parse(std::vector<int64_t>& x, const std::string& param_name) -> void {
```

Comment:
Why is there a new `TupleParser` function. It looks like you are only removing uses of `TupleParser`?